crates/oxc_minifier/src/ast_passes/mod.rs

```diff
@@ -39,3 +39,51 @@ pub trait CompressorPass<'a>: Traverse<'a> {
 
     fn build(&mut self, program: &mut Program<'a>, ctx: &mut TraverseCtx<'a>);
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use oxc_allocator::Allocator;
+    use oxc_ast::ast::Statement;
+    use oxc_parser::Parser;
+    use oxc_semantic::SemanticBuilder;
+    use oxc_span::SourceType;
+
+    #[derive(Default)]
+    pub struct Tester;
+
+    fn build<'a>(allocator: &'a Allocator, source_text: &'a str) -> (TraverseCtx<'a>, Program<'a>) {
+        let source_type = SourceType::mjs();
+
+        let program = Parser::new(allocator, source_text, source_type).parse().program;
+        let (symbols, scopes) =
+            SemanticBuilder::new().build(&program).semantic.into_symbol_table_and_scope_tree();
+
+        (TraverseCtx::new(scopes, symbols, allocator), program)
+    }
+
+    fn get_string_literal(source_text: &str) -> Option<String> {
+        let allocator = Allocator::default();
+        let (ctx, program) = build(&allocator, source_text);
+
+        let Some(Statement::ExpressionStatement(expr_stmt)) = program.body.first() else {
+            return None;
+        };
+
+        ctx.get_string_literal(&expr_stmt.expression).map(Into::into)
+    }
+
+    #[test]
+    fn test_get_string_literal() {
```

Comment:
I don't know how to implement the test template and where to place it.

Reply:
I have implemented a simple test case and temporarily placed it here.

Reply:
@camc314 cc

Reply:
I would recommend you to put it to `src/node_util/mod.rs` instead.

Reply:
To be honest, I'm not sure if it's still necessary to exist. Because our `TemplateLiteral` seems to be able to be compressed into `StringLiteral`. I don't know if there are any specific scenarios that require this method.